bitcoinplusplus/layer-2/exploring-p2p-hashrate-markets-v2.md

```diff
@@ -218,44 +236,52 @@ And cloud mining, you're getting price gouged almost always.
 And again, this kind of goes back to, I think, what I had mentioned before, is that why are we going to these traditional financial institutions looking for money when they're the ones that like, usually, one, they don't understand Bitcoin, they have no real incentive to understand or try and take that risk on Bitcoin outside of the fact that they're gonna try and be relatively predatory on the fact that, oh, you need the money and no one else is gonna give you the money, so I'm gonna give it to you for like really expensively.
 And yeah, it creates this situation, right, where nobody's really out trying to buy these future hash rate contracts in the traditional financial markets.
 Other considerations are, well, what if you want to actually take delivery?
 There's nothing really like that.
 What about the duration of this contract?
-So for the most part, if you look at marketplaces like NiceHash and Mining Regret tools, you have pretty short contract durations there, which makes it okay if you're kind of trying to do the betting in one direction or another, but it's fairly short, right?
+So for the most part, if you look at marketplaces like NiceHash and Mining Regret tools, you have pretty short contract durations there, which makes it okay if you're kind of trying to do the betting in one direction or another, but it's fairly short.
 So you're not really buying something and then just kind of letting it sit there and do its thing.
 And then in terms of the actual non-deliverables or like the swaps, like contracts, well, it's, you know, you're buying kind of like a traditional financial instrument and it's not code.
 Oh, wow, okay.
 That was fast.
 Well, I'm gonna run through this really quick.
 Thank you.
+
+
+## P2P Hashrates
+
 So our model is where you go, we have auctions set up in order to have price discovery.
 We have multi-sig where the payment to the seller of the hash rate, or the miner, only occurs after the hash rate's been delivered, and we deliver the hash rate through a proxy.
 So we're able to see exactly how much hash rate ends up getting delivered.
 So Here we have a little model of how this works.
 So we create this little agreement here in the middle.
 The agreement, the purchase part here in orange at the bottom is held on chain in multisig.
 The delivery for The terms of delivery are held through the pool account.
-So the hash rate gets pointed here from the miner to a pool where eventually a coinbase hits the pool and the payout goes to a mining pool account, right?
+So the hash rate gets pointed here from the miner to a pool where eventually a coinbase hits the pool and the payout goes to a mining pool account.
 And that's where this part of the agreement comes into play.
-And finally, you can actually obviously remove your funds from your account model to the next model, right?
+And finally, you can actually obviously remove your funds from your account model to the next model.
 So you have your hot dog shape.
```

Comment:
It seems there might be some uncertainty about the phrase "hot dog shape" in this part of the transcript. If the audio isn't clear enough to confirm what the speaker is saying, it's perfectly acceptable to use "[inaudible]" to indicate segments that are difficult to hear or understand. This ensures our transcript remains accurate and reliable, even when facing audio challeng
```suggestion
And finally, you can actually obviously remove your funds from your account model to the next model, so you have your [inaudible] shape.
```